aiormq/connection.py

```diff
@@ -532,6 +537,25 @@ async def __reader(self, frame_receiver: FrameReceiver) -> None:
                     continue
                 elif isinstance(frame, spec.Channel.CloseOk):
                     self.channels.pop(channel, None)
+                elif isinstance(frame, spec.Connection.UpdateSecretOk):
+                    if (
+                        self.__update_secret_future is not None and
+                        not self.__update_secret_future.done()
+                    ):
+                        self.__update_secret_future.set_result(frame)
+                    else:
+                        log.warning("Got unexpected UpdateSecretOk frame")
+                    continue
+                elif isinstance(frame, spec.Connection.Blocked):
+                    log.warning(
+                        "Connection %r was blocked by: %r", self, frame.reason
+                    )
+                    self.__connection_unblocked.clear()
+                    continue
+                elif isinstance(frame, spec.Connection.Unblocked):
+                    log.warning("Connection %r was unblocked", self)
```

Comment:
may be make it lower? debug e.g.

Reply:
That’s really important, that’s means the broker is recovery.